File: lib/Vector.py
class Vector2:
    def __init__(x= 0 :float, y=0 : float):
        self.x = x;
        self.y = y;

    def __add__(self, b):
        return Vector2(self.x+b.x, self.y+b.y)

    def __len__(self):
        return sqrt(self.x**2+self.y**2)

File: lib/Vector.ts

class Vector2 {
    x: number
    y: number
    constructor(x: number = 0, y: number = 0) {
        this.x = x
        this.y = y

    }

    get length(): number {
        return Math.sqrt(this.x ** 2 + this.y ** 2)
    }
    
    add(target: Vector2 ){
        this.x += target.x;
        this.y += target.y;
    }
}